### ISSUE TITLE
iGPU UHD630 support?

### ISSUE THREAD

**Issue by mdevDE** [SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3#2]
Hi and thank you for your great work. Please keep up the good work.

I just wonder if it is possible to use only the internal gpu without a dedicated gpu like the 5700 XT and how this can. be achieved.

Thanks in advance

**Comment by fidalgodev** [comment]
Yes. Read the Open Core guides. You need to remove the related RX5700 SSDT's and Kext's files from the folders and from the config.plist file, and you also need to change a property on the config.plist:

`AAPL,ig-platform-id` from `0300C89B` to `07009B3E`

Read more [here](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html) (search for the AAPL,ig-platform-id field and you will find the info)

**Comment by SchmockLord** [comment]
The device properties don't need to be changed. I have already setup the config_iMac20,2.plist in a way where the iGPU is enabled for display output. So with '''07009B3E''' and all the framebuffer settings.

But as @fidalgodev said, you need to remove the SSDT-5700XT.aml and DAGPM.kext from your config.

**Comment by mdevDE** [comment]
Thank you both for the very fast responses!

**Comment by fidalgodev** [comment]
@SchmockLord I see you updated this part:

```
device-id=0xA170
The layout-id and the device-id is injected via the device properties.
```

Is the device-id no longer needed on `PciRoot(0x0)/Pci(0x1F,0x3)` ? 

Asking because it's still on the samples configs

**Comment by freber** [comment]
I'm also curious if someone got 4K screens working with iGPU, currently don't get full resolution.

**Comment by lctncld** [comment]
@freber I was able to get both 4k screens to work with `AAPL,ig-platform-id=00009B3E`. Actually my system didn't boot at all without this value. One screen is connected to DP port and another one to a TB3 dock. 
I got the value from the [Desktop Comet Lake](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/coffee-lake.html#deviceproperties) page of OpenCore install guide.

<img width="832" alt="Screenshot 2020-11-13 at 09 20 14" src="https://user-images.githubusercontent.com/2962997/99101435-76e2e900-2591-11eb-877f-561731875a5f.png">
